lib/fluent/plugin/out_dynatrace.rb

```diff
@@ -64,7 +64,12 @@ def configure(conf)
         compat_parameters_convert(conf, :inject)
         super
 
+        raise Fluent::ConfigError, 'active_gate_url has not been set' if @active_gate_url == ''
+
         @uri = URI.parse(@active_gate_url)
+        raise Fluent::ConfigError, 'active_gate_url scheme must be http or https' unless
+          %w[http https].include?(@uri.scheme)
+
         @agent = Net::HTTP.new(@uri.host, @uri.port)
 
```

Comment:
Please bump the version in https://github.com/dynatrace-oss/fluent-plugin-dynatrace/blob/main/lib/fluent/plugin/dynatrace_constants.rb to `0.2.0`.